### PR TITLE
Document new settings for TCP listener(s) and IPv6 support

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

Documenting the new settings we are planning to include in the next version, namely:

- `TCP Server Address` and `TCP Server Port`
  - To configure the listener(s) to generate TCP candidates.
- `Enable IPv6`
   - To enable experimental support for dual-stack connectivity.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-50294
https://mattermost.atlassian.net/browse/MM-52268

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]
Thank you, @streamer45! Great docs update!

File: source/configure/plugins-configuration-settings.rst [hunk 1]
```diff
@@ -507,42 +507,68 @@ Enable plugin
...
-~~~~~~~~~~~~~~~
+RTC server port (UDP)
+~~~~~~~~~~~~~~~~~~~~~
 
 .. include:: ../_static/badges/selfhosted-only.rst
   :start-after: :nosearch:
 
 +-----------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------+
-| This setting controls the UDP port listened on by the RTC server. All Calls traffic will be served through this port. | - System Config path: **Plugins > Calls**                                                              |
+| This setting controls the UDP port listened on by the RTC server. Calls traffic will be served through this port.     | - System Config path: **Plugins > Calls**                                                              |
```
Comment (cpoile) [inline source/configure/plugins-configuration-settings.rst:571]: Maybe?

```suggestion
| This setting controls the UDP port listened on by the RTC server. All UDP Calls traffic will be served through this port.     | - System Config path: **Plugins > Calls**                                                              |
```

File: source/configure/plugins-configuration-settings.rst [hunk 2]
```diff
@@ -552,6 +578,35 @@ RTC server port
 | **Note**: This setting is only applicable when not running calls through the standalone ``rtcd`` service.             |                                                                                                        |
 +-----------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------+
 
+.. config:setting:: plugins-callsrtcserverporttcp
+  :displayname: RTC server port (TCP) (Plugins - Calls)
+  :systemconsole: Plugins > Calls
+  :configjson: PluginSettings.Plugins.com.mattermost.calls.tcpserverport
+  :environment: N/A
+  :description: The TCP port the RTC server will listen on. Calls traffic will be served through this port. Default port is **8443**.
```
Comment (cpoile) [inline source/configure/plugins-configuration-settings.rst:586]: Maybe?

```suggestion
  :description: The TCP port the RTC server will listen on. All TCP Calls traffic will be served through this port. Default port is **8443**.
```

File: source/configure/plugins-configuration-settings.rst [hunk 3]
```diff
@@ -507,42 +507,68 @@ Enable plugin
...
+| - This setting is available starting in plugin version 0.17.                                                                       |                                                                                                           |
++------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
+
+.. config:setting:: plugins-callsrtcserverportudp
+  :displayname: RTC server port (UDP) (Plugins - Calls)
   :systemconsole: Plugins > Calls
   :configjson: PluginSettings.Plugins.com.mattermost.calls.udpserverport
   :environment: N/A
-  :description: The UDP port the RTC server will listen on. All calls traffic will be served through this port. Default port is **8443**.
+  :description: The UDP port the RTC server will listen on. Calls traffic will be served through this port. Default port is **8443**.
```
Comment (cpoile) [inline source/configure/plugins-configuration-settings.rst:562]: Maybe?

```suggestion
  :description: The UDP port the RTC server will listen on. All UDP Calls traffic will be served through this port. Default port is **8443**.
```

File: source/configure/plugins-configuration-settings.rst [hunk 4]
```diff
@@ -552,6 +578,35 @@ RTC server port
...
+  :description: The TCP port the RTC server will listen on. Calls traffic will be served through this port. Default port is **8443**.
+
+RTC server port (TCP)
+~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/selfhosted-only.rst
+  :start-after: :nosearch:
+
++-----------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------+
+| This setting controls the TCP port listened on by the RTC server. Calls traffic will be served through this port.     | - System Config path: **Plugins > Calls**                                                              |
```
Comment (cpoile) [inline source/configure/plugins-configuration-settings.rst:595]: ?
```suggestion
| This setting controls the TCP port listened on by the RTC server. All TCP Calls traffic will be served through this port.     | - System Config path: **Plugins > Calls**                                                              |
```

**Review by cpoile — APPROVED** [review]
Just minor suggestions, maybe it's not important to be _that_ specific, but maybe it's clearer?

**Comment by cwarnermm** [comment]
@streamer45 - Is this docs PR intended for the v8.0 release?

**Comment by streamer45** [comment]
> @streamer45 - Is this docs PR intended for the v8.0 release?

@cwarnermm Yes, they can be merged any time I suppose.

**Pull request merged**